docker/build/cicd.ubuntu.Dockerfile

```diff
@@ -15,9 +15,9 @@ ENV GOROOT=/usr/local/go \
 RUN mkdir -p $GOPATH/src/github.com/algorand
 COPY . $GOPATH/src/github.com/algorand/go-algorand
 ENV PATH=$GOPATH/bin:$GOROOT/bin:$PATH \
-    GOPROXY=https://pkg.go.dev
+    GOPROXY=https://proxy.golang.org,https://pkg.go.dev
```

Comment:
If the goproxy needs to be updated here, wouldn't it need to be updated everywhere else as well ?

Reply:
Yes, this is just me testing things separately. I'll shift this back to draft.

Reply:
No worries. Its perfectly fine to do a partial fix in one pr, and follow up in another.